.github/workflows/ingestion-functions-deploy.yml

```diff
@@ -34,3 +35,15 @@ jobs:
         docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -t $ECR_REGISTRY/$ECR_REPOSITORY .
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+
+    - name: Build, tag, and push image to Amazon ECR
+      if: ${{ github.ref == 'refs/heads/1.6-stable' }}
```

Comment:
This either wants to be fuzzy matched too, or we need a variable somewhere that says what our current stable version is. Or we need a long-lived stable branch. Whichever way, we don't want to fix "Batch is on main, prod is on stable" by introducing "Batch is on old stable, prod is on new stable".